packages/components/src/components/data-table/_data-table-expandable.scss

```diff
@@ -92,6 +94,15 @@
     box-shadow: none; // first td doesn't have a visible border
   }
 
+  .#{$prefix}--child-row-inner-container h1 {
```

Comment:
Are these selectors guaranteed to be `h1` and `p`? Is there a class we could hook into instead?

Reply:
@joshblack I did have a question about this since I did prefer a class. The way its written is there a `--child-row-inner-container` and the children in it are the h1 and p with no classes. Should I create new classes ?